src/main/groovy/jaci/gradle/deploy/artifact/AbstractArtifact.groovy

```diff
@@ -20,6 +20,7 @@ abstract class AbstractArtifact implements Artifact, Configurable<Artifact> {
 
     private disabled = false
 
+    @Inject
```

Comment:
What are we injecting?

Reply:
@Inject is needed for anything created with gradle's `create` or `newInstance`, which is needed for Action decoration to be injected. The artifacts we provide here don't really use it, but a downstream artifact might, and its better to support that then not. The target was the only one that actually needed this, but I figured do it for all of them.